pkg/server/router/manifest.go

```diff
@@ -182,3 +184,242 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 
 	return framework.Respond(ctx, w, nil, http.StatusOK)
 }
+
+type SubmitApplicationRequest struct {
+	ManifestID             string                              `json:"manifestId" validate:"required"`
+	PresentationSubmission exchangensdk.PresentationSubmission `json:"presentationSubmission" validate:"required"`
+}
+
+func (sar SubmitApplicationRequest) ToServiceRequest() manifest.SubmitApplicationRequest {
+	return manifest.SubmitApplicationRequest{
+		PresentationSubmission: sar.PresentationSubmission,
+		ManifestID:             sar.ManifestID,
+	}
+}
+
+type SubmitApplicationResponse struct {
```

Comment:
this is fine...but I'm wondering if wrapping it in a new object with a label of `credential_response` may be better...I'm leaning towards no and should keep it as is

Reply:
Yea and I think this will change very quickly when we wrap it in a VP